### ISSUE TITLE
Encoding cancellations

### ISSUE THREAD

**Issue by karljonaskarlsson** [BetaMasaheft/Documentation#2243]
I think there is a need for a way of encoding cancellations by encircling or under-/overlining. This seems not to be covered by the Guidelines (https://betamasaheft.eu/Guidelines/?id=text-encoding).

Examples of what I mean:

<img width="401" alt="Skärmavbild 2022-09-27 kl  14 10 54" src="https://user-images.githubusercontent.com/34240156/192523057-ec6a3356-f9b1-4041-87ef-6cc033c5bcd6.png">

**Comment by thea-m** [comment]
It is actually currently covered by the definition of "expunctuated" given in the schema: "marked with dots above or below the line, marked with lines above and below or encircled". I have no objection against splitting this into three values for the element `<del>`, such as:

- expunctuated = "marked with dots above or below the line"
- encircled = "encircled"
- underOverlined = "under-/overlining"

If no one disagrees, I can edit the schema. But note that we will not go through all existing cases marked up with "expunctuated" to update them, so there will be imprecisions left. I can live with that.

**Comment by thea-m** [comment]
https://betamasaheft.eu/Guidelines/?id=del

**Comment by karljonaskarlsson** [comment]
I think adding more detail to the description of this phenomenon is a good idea! What you suggests sounds good (and then a HiWi will go through all the older examples of "expunctuation" and check them in 2031).

**Comment by eu-genia** [comment]
(and do not forget that changing the schema is not enough (I guess you are only speaking about the `@rend` attribute?) if you want to see your encoding results in the app)

**Comment by thea-m** [comment]
@DenisNosnitsin1970 @karljonaskarlsson how would you like these values (expunctuated, encircled, over/underlined) to be visualised in the app?

**Comment by thea-m** [comment]
From @eu-genia in the PR:

> we can only try to implement what html may achieve - and what does not coincide with anything else
> 
> one can have borders, dotted borders, underlines, dotted underlines, as long as they have no other meanings assigned
> 
> at the moment i believe there is no visualization for del rend="expunctuated" whatsoever

@DenisNosnitsin1970 @karljonaskarlsson what would be your suggestions for the visualisation?

**Comment by DenisNosnitsin1970** [comment]
To get it right - it is about how an expunctuated text will be visualised in the original text typed, right?

**Comment by thea-m** [comment]
How it will be visualised on the website

**Comment by DenisNosnitsin1970** [comment]
OK, I see, it is justified question but in no way a very easy one. Probably we have two ways: 1) we can invent smth (like we say: expunctuated bits can be marked by dots around), 2) we need to look up if there are  any editorial means for that already used in philological practice or elsewhere.

**Comment by karljonaskarlsson** [comment]
Is visualisation on the website always in the form of signs, or are "sic" and other Latin terms used?

**Comment by thea-m** [comment]
There is no rule regarding this.

**Comment by karljonaskarlsson** [comment]
Then one alternative could be to use Latin (or English?) terms in some kind of brackets. If you think that this is suitable. For me, it does not really matter. Also, it would be ok for me if they are encoded differently but visualised the same.

**Comment by karljonaskarlsson** [comment]
@eu-genia , can I close this issue or is it better to keep it open?

**Comment by eu-genia** [comment]
Was there any solution? Any specific suggestion to change to the schema and visualization? If the issue is valid something must be done before it can be closed I think but I guess I never had a clear instruction

**Comment by karljonaskarlsson** [comment]
I believe that Denis edited the schema (because the values he suggested, i.e. "expunctuated, encircled, over/underlined" are currently present), but it seems like no solution was reached regarding how this should be visualised in the app. As the discussion has fallen asleep, I think it would make sense to close this issue for now, and if someone wants to revisit the topic, they can open a new issue about the visualisation.

**Comment by eu-genia** [comment]
it was Dorothea who edited the schema, it is visible in the issue history, but there was nothing done for the app as nothing has been proposed. the app shows all letters marked as deleted with no circles or lines or dots

**Comment by karljonaskarlsson** [comment]
@DenisNosnitsin1970 @thea-m , do we want to reactivate the question about the visualisation of these features? Or are we happy to leave it as it is for the moment?

**Comment by DenisNosnitsin1970** [comment]
I think now it is not a priority, can be left for the future. But I would not close this, if possible; to remember this again will be difficult.

**Comment by eu-genia** [comment]
expunctuated 

![Image](https://github.com/user-attachments/assets/3c4a27e3-1118-464c-a025-e8d06a12eaa3)

encircled

![Image](https://github.com/user-attachments/assets/0e5d1033-473e-48ea-ab69-5d74bc5a9b8f)

overUnderlined

![Image](https://github.com/user-attachments/assets/5e1b782a-bdf5-4735-9fb1-3b26b275690c)

effaced = gray
